crates/apub/src/objects/community.rs

```diff
@@ -187,11 +187,9 @@ impl Object for ApubCommunity {
     let context_ = context.reset_request_count();
     spawn_try_task(async move {
       group.outbox.dereference(&community_, &context_).await.ok();
-      group
-        .followers
-        .dereference(&community_, &context_)
-        .await
-        .ok();
+      if let Some(followers) = group.followers {
+        followers.dereference(&community_, &context_).await.ok();
```

Comment:
I'm not familiar with the `dereference` function and I couldn't find docs for it. I figure it's related to the dereference operator `*`?

Reply:
Probably from a trait in the activitypub_federation crate

Reply:
https://docs.rs/activitypub_federation/latest/activitypub_federation/index.html#fetching-data